### ISSUE TITLE
[Bug]: Playwright.CLI 1.2.1 fails during install command on macOS

### ISSUE THREAD

**Issue by martincostello** [microsoft/playwright-dotnet#1893]
### Playwright version

1.2.1

### Operating system

MacOS

### Other information

.NET 6.0.101 SDK on a GitHub Actions hosted macOS agent (also fails with 6.0.100).

```
Current runner version: '2.285.1'
Operating System
  macOS
  11.6.1
  20G224
Virtual Environment
  Environment: macos-11
  Version: 20211207.1
  Included Software: https://github.com/actions/virtual-environments/blob/macOS-11/20211207.1/images/macos/macos-11-Readme.md
  Image Release: https://github.com/actions/virtual-environments/releases/tag/macOS-11%2F20211207.1
```

### What happened? / Describe the bug

Running `playwright install` on macOS fails with version 1.2.1.

```sh
  Unhandled exception. Microsoft.CSharp.RuntimeBinder.RuntimeBinderException: Cannot perform runtime binding on a null reference
     at CallSite.Target(Closure , CallSite , Object , String[] )
     at System.Dynamic.UpdateDelegates.UpdateAndExecuteVoid2[T0,T1](CallSite site, T0 arg0, T1 arg1)
     at Microsoft.Playwright.CLI.Program.Main(String[] args) in /_/src/Playwright.CLI/Program.cs:line 83
  /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmp5910ae2db37a47bab05fc6261d59e372.exec.cmd: line 2:  1361 Abort trap: 6           playwright install
/Users/runner/work/dotnet-playwright-tests/dotnet-playwright-tests/PlaywrightTests/PlaywrightTests.csproj(16,5): error MSB3073: The command "playwright install" exited with code 134.
```

This is occurring in all of my GitHub repos where I install the latest version of the Playwright.CLI global tool using the command `dotnet tool install --global Microsoft.Playwright.CLI` on macOS. It does not affect Linux or macOS.

Here is one such example failing job is [here](https://github.com/martincostello/dotnet-playwright-tests/runs/4529975610?check_suite_focus=true#step:5:14).

Explicitly installing version `1.2.0` fixes it: `dotnet tool install --global Microsoft.Playwright.CLI --version 1.2.0` ([example](https://github.com/martincostello/dotnet-playwright-tests/pull/30)).

### Code snippet to reproduce your bug

```shell
N/A
```

**Comment by martincostello** [comment]
Interesting - on GitHub Actions I wasn't getting this issue with Windows or Linux 🤔

**Comment by martincostello** [comment]
Looks like the same issue was also reported in #1892 while I was writing up this issue.

**Comment by mxschmitt** [comment]
Some summary:

- Can be reproduced on windows-latest and ubuntu-latest on GitHub Actions
- On some machines it does only repro on Windows
- On this PR, I can repro on MacOS only: https://github.com/microsoft/playwright-dotnet/runs/4528734255?check_suite_focus=true
- Fails on .NET 6.0.101 SDK and .NET 6.0.100 SDK

Looking into it...

**Comment by mxschmitt** [comment]
@slang25 @martincostello @wozzo

We don't recommend the `Microsoft.Playwright.CLI` as a tool anymore, because it side-effects when you have multiple Playwright's of different languages installed since its installed globally. The new way is via the PowerShell script, see in docs: https://playwright.dev/dotnet/docs/intro#first-project

(we still fix the regression soon)

**Comment by wozzo** [comment]
Cheers. Will update to new approach tomorrow.

**Comment by mxschmitt** [comment]
We published 1.2.2 of `Microsoft.Playwright.CLI` which should fix the issues.

**Comment by martincostello** [comment]
Confirmed working again as before with 1.2.2 on all three operating systems in GitHub Actions. Thanks for the quick turnaround.
